ext/native.go

```diff
@@ -569,9 +667,22 @@ func (t *nativeType) Value() any {
 	return t.typeName
 }
 
+// fieldByName returns the corresponding reflect.StructField for the give name either by matching
+// field tag or field name.
+func (t *nativeType) fieldByName(fieldName string) (reflect.StructField, bool) {
+	for i := 0; i < t.refType.NumField(); i++ {
```

Comment:
If we already have the `t.parseStructTags` info, then it's worth early returning with `t.refType.FieldByName(fieldName)`